cob_twist_controller/cfg/TwistController.cfg

```diff
@@ -7,25 +7,32 @@ gen = ParameterGenerator()
 
 damping_method_enum = gen.enum([ 
                        gen.const("manipulability",      int_t, 0, "manipulability"),
-                       gen.const("constant",            int_t, 1, "constant"),
-                       gen.const("truncation",          int_t, 2, "truncation")],
+                       gen.const("constant",            int_t, 1, "constant"),], 
```

Comment:
Can we have the following damping_methods?
None (=0) -> damping_factor = 0.0 konstant
Constant (=1) -> damping_factor = damping_factor dynamic_reconfigure param
Manipulability (=2) -> damping_factor = f(manipulability)

Then we would be consistent with wln_constraints (from disabled to more advanced technique)

Then we would